.github/workflows/scorecards-analysis.yml

```diff
@@ -1,9 +1,8 @@
 name: Scorecards supply-chain security
 on:
-  # Only the default branch is supported.
-  branch_protection_rule:
   push:
-    branches: [ main ]
+    branches:
+    - main
```

Comment:
Aren't these just different syntax for the same thing? Or is the fix the removal of the empty entry above?

Reply:
Yeah not sure it the empty branch protection was causing this workflow to run on push of any branch.